server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -292,3 +292,31 @@ watermark_module_opened=false
 watermark_secret_key=IoTDB*2019@Beijing
 watermark_bit_string=100101110100
 watermark_method=GroupBasedLSBMethod(embed_row_cycle=2,embed_lsb_num=5)
+
+
+####################
+### Configurations for creating schema automatically
+####################
+
+# Is automatically creating schema enable
+enable_auto_create_schema=true
+
+# Storage group level when creating schema automatically is enable
+# The default is 2
```

Comment:
add an example, such as root.sg1.d1.s1. when this level is 1, we set root.sg1 as a storage group